RELEASE_NOTES.txt

```diff
@@ -1,6 +1,88 @@
 Apache Ignite Release Notes
 ===========================
 
+
+Apache Ignite In-Memory Database and Caching Platform 2.8.1
+-----------------------------------------------------------
+
+Security:
+* Fixed vulnerability of using H2 admin rights for user SQL connections
+
+Ignite Core:
+* Added transaction operations metrics
```

Comment:
I'd suggest sorting these lines in alphabetical order (e.g. in the IntelliJ IDEA `Edit -> Sort lines`).

Reply:
fixed.